app/jobs/regular/pull_hotlinked_images.rb

```diff
@@ -88,12 +88,14 @@ def execute(args)
         # we never want that job to bump the topic
         options = { bypass_bump: true }
         post.revise(Discourse.system_user, changes, options)
+      elsif downloaded_urls.present?
+        post.trigger_post_process(true)
       end
     end
 
     def extract_images_from(html)
       doc = Nokogiri::HTML::fragment(html)
-      doc.css("img[src]") - doc.css(".onebox-result img") - doc.css("img.avatar")
```

Comment:
Yes I was. I thought all avatar images are always downloaded locally. But now I realize SSO overridden avatars will have external URLs.

I guess it should be included again. Am I right?